content/v2.0/influxdb-templates/use.md

```diff
@@ -161,6 +162,21 @@ influx pkg \
   --recurse
 ```
 
+### Define environment references
+Some templates include [environment references](/v2.0/influxdb-templates/create/#include-user-definable-resource-names) that let you provide custom names for resources in the template.
+Template maintainers should document what environment references exist in the template
```

Comment:
If you use environment refs, document the env refs available in your template and list the keys...